site/2016/02/16/doctrine-mongodb-odm-release-1.0.5.rst

```diff
@@ -0,0 +1,43 @@
+Doctrine MongoDB ODM release 1.0.5
+==================================
+
+We are happy to announce the immediate availability of Doctrine MongoDB ODM
+`1.0.5 <https://github.com/doctrine/mongodb-odm/releases/tag/1.0.5>`__.
+
+Bug fixes in this release
+-------------------------
+
+Notable fixes may be found in the
+`changelog <https://github.com/doctrine/mongodb-odm/blob/master/CHANGELOG-1.0.md#105-2016-02-16>`__.
+A full list of issues and pull requests included in this release may be found in the
+`1.0.5 milestone <https://github.com/doctrine/mongodb-odm/issues?q=milestone%3A1.0.5>`__.
+
+Installation
+------------
+
+You can install the latest version using the following ``composer.json`` definitions:
+
+.. code-block:: json
+
+  {
+      "require": {
+          "doctrine/mongodb-odm": "^1.0.5"
+      }
+  }
+
+Doctrine MongoDB ODM and PHP 7
+------------------------------
+
+While ODM still relies on old Mongo driver and no dates are scheduled for the 2.0 release,
+`it is already possible to run ODM with PHP 7! <https://twitter.com/alcaeus/status/697659616172359680>`__
+(please note that for now you need to be running ``dev-master`` version of both ODM and the underlying
+``doctrine/mongodb`` library, this will change as soon as we release 1.1 and 1.3 versions respectively).
+This is all possible thanks to our Andreas Braun (`@alcaeus <https://twitter.com/alcaeus>`__) work on
+`mongo-php-adapter <https://github.com/alcaeus/mongo-php-adapter>`__ which provides ext-mongo interface
+using new `mongo-php-library <https://github.com/mongodb/mongo-php-library>`__. If you can't wait to
+give ODM a test flight on PHP 7 now is the time! Also if you meet Andreas be sure to buy him a beer :)
```

Comment:
Suggestion (feel free to add links, of course): "`mongo-php-adapter`, which implements `ext-mongo`'s API atop `ext-mongodb` and `mongodb-php-library`."